README.md

```diff
@@ -1,3 +1,103 @@
 # valletta
 
 Tool to build and manage [Discourse](https://discourse.org) theme customizations. Generate `.json` files that you can upload or paste into the Discourse admin interface, while you can make the most of your existing workflow and source control.
```

Comment:
I guess we can break the lines here, since the rest of the new document is word-wrapped manually, so to be consistent?